staff/marc-salas/precurse/Bingo/bingo.js

```diff
@@ -0,0 +1,144 @@
+    // Declaración de variables globales
+    var carton =[[],[],[]];
+    var numerosCantados = [];
+    var random;
+    var bingoDone = false;
+    var fin = true;
+    var cont = true;
+    var jugador;
+    var linDone = false;
+    var turnos = 0;
+    // Preguntar nombre del jugador y presentación
+    function nombre(){
+        jugador = prompt('Introduzca su nombre','');
+        console.log('buenos dias senyor/a '+ jugador + ', este es su carton!') 
+    }
+    // Generar el carton 
+    function generarCarton(){
+        let y = 0;
+        while (y < 3){
+            let x = 0;
+            while (x < 5) {
+                var aleatori = Math.floor(Math.random() * 99) + 1;
```

Comment:
Good Evening Marc, another way to do a bidimensional array inside function "generarCarton" without using whiles is : `var x = new Array(3);
for (var i = 0; i < x.length; i++) {
  x[i] = new Array(5);
}
console.log(x);`

Reply:
That's a good comment !